code/game/jobs/whitelist.dm

```diff
@@ -1,24 +1,3 @@
-#define WHITELISTFILE "data/whitelist.txt"
-
-GLOBAL_LIST_EMPTY(whitelist)
-
-/proc/init_whitelists()
```

Comment:
This stuff is a bay12 carryover for whitelisting ckeys to certain species. The init is never called here, and its dead code.